### ISSUE TITLE
Empty data in Chosen Positions are not handled corretly.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halalaninitiative/halalan#3]
```
What steps will reproduce the problem?
1. Go to Add Voter.
2. Don't enter any data.
3. Press submit.

What is the expected output? What do you see instead?
Chosen Positions should be empty but the data in Possible Positions are
copied to it.
```

Original issue reported on code.google.com by `waldemar...@gmail.com` on 11 Feb 2008 at 3:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `waldemar...@gmail.com` on 11 Feb 2008 at 3:26
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `toni...@gmail.com` on 16 Feb 2008 at 12:05
- Changed state: **Verified**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 44a4b32c1d.
```

Original comment by `waldemar...@gmail.com` on 4 Sep 2010 at 1:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `aicapa...@gmail.com` on 7 Feb 2011 at 9:19
- Changed state: **Verified**
